Source/OxyPlot/Series/DataRange.cs

```diff
@@ -0,0 +1,111 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DataRange.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Represents an interval defined by two doubles
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Series
+{
+    using System;
+
+    /// <summary>
+    /// Represents an interval defined by two doubles.
+    /// </summary>
+    public struct DataRange : ICodeGenerating
+    {
+        /// <summary>
+        /// The undefined data range.
+        /// </summary>
+        public static readonly DataRange Undefined = new DataRange(double.NaN, double.NaN);
+
+        private double? minimum;
+        private double? maximum;
```

Comment:
- Should be `readonly`.
 - I'd prefer these were non-nullable, and there was an explicit `isDefined` field, and suitable validation was performed in the constructor, but it's not a big deal

Reply:
Done.